OpenSim/Tools/Test/test_wrapAllVis.osim

```diff
@@ -0,0 +1,745 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Where is model being used? I could not find it in the visualization tests. By its name it looks like it is to be a test for visualizing wrapping, which maybe slipped in from a future PR?